extensions/ql-vscode/test/unit-tests/common/file-path-set.test.ts

```diff
@@ -0,0 +1,58 @@
+import { FilePathSet } from "../../../src/common/file-path-set";
+
+describe("FilePathSet", () => {
+  describe("isEmpty", () => {
+    it("should return true only when set is empty", () => {
+      const v = new FilePathSet();
+      expect(v.isEmpty()).toBe(true);
+
+      v.addPath("/foo");
+      expect(v.isEmpty()).toBe(false);
+
+      v.popPath();
+      expect(v.isEmpty()).toBe(true);
+    });
+  });
+
+  describe("addPath / popPath", () => {
+    it("should keep all paths when they don't overlap", () => {
+      const v = new FilePathSet();
+      v.addPath("/foo");
+      v.addPath("/bar");
+      v.addPath("/baz");
+      expect(v.popPath()).toBe("/foo");
+      expect(v.popPath()).toBe("/bar");
+      expect(v.popPath()).toBe("/baz");
+      expect(v.popPath()).toBe(undefined);
```

Comment:
The documentation of `popPath` doesn't specify an order, while the tests do depend on it. Should we be making explicit that paths are popped in insertion-order, or should we change the tests so that they pop all paths and then check the result?

Reply:
This file has been deleted, so no need to do anything here now.